pyresample/bilinear/xarr.py

```diff
@@ -357,18 +360,21 @@ def _mask_coordinates_dask(lons, lats):
 
 
 def _get_bounding_corners_dask(in_x, in_y, out_x, out_y, neighbours, idx_ref):
-    """Get four closest locations from (in_x, in_y) so that they form a
+    """Get bounding corners.
+
+    Get four closest locations from (in_x, in_y) so that they form a
     bounding rectangle around the requested location given by (out_x,
     out_y).
     """
-
     # Find four closest pixels around the target location
 
     # FIXME: how to daskify?
     # Tile output coordinates to same shape as neighbour info
     # Replacing with da.transpose and da.tile doesn't work
-    out_x_tile = np.transpose(np.tile(out_x, (neighbours, 1)))
-    out_y_tile = np.transpose(np.tile(out_y, (neighbours, 1)))
+    out_x_tile = np.reshape(np.tile(out_x, neighbours),
+                            (neighbours, out_x.size)).T
```

Comment:
Have you checked the dask source to see if this is intentionally not implemented (meaning, non-trivial)? Or did someone just not implement it yet and you could?

Reply:
No, didn't have time, just made it to work :grin:

Reply:
The source doesn't mention if this omission is by design or just a missing feature (https://github.com/dask/dask/blob/master/dask/array/creation.py#L802), but there's an issue from 2017 about this feature with new comments from June this year with a suggested solution: https://github.com/dask/dask/issues/2447

Reply:
We probably mentioned this on slack, but do you remember what the error was or what was wrong when using the dask version?

Reply:
Dask raises a `NotImplementedError` when trying to `tile` with a `tuple`. The call to `np.tile()` calls the dask version in the background for dask arrays and causes the error to be raised.